test/htmltojsx-test.js

```diff
@@ -136,6 +136,42 @@ describe('htmltojsx', function() {
       expect(converter.convert('<div style="font-size: 12pt">Test</div>').trim())
         .toBe('<div style={{fontSize: \'12pt\'}}>Test</div>');
     });
+	
+	it('should convert vendor-prefix "style" attributes', function() {
```

Comment:
Alright, hopefully that fixes the last of it.  Do you have some linting rules that could be put in here?